src/Orchard.Web/Modules/Orchard.Layouts/Controllers/BlueprintAdminController.cs

```diff
@@ -43,12 +44,19 @@ public class BlueprintAdminController : Controller, IUpdateModel {
             _shapeFactory = shapeFactory;
             _transactionManager = transactionManager;
             _signals = signals;
+            Services = orchardServices;
+
             T = NullLocalizer.Instance;
         }
 
+        public IOrchardServices Services { get; set; }
```

Comment:
I see. So it would be automatically be set using property injection. Which is kind of cool. The downside I can think of is that the dependency on `IOrchardServices` becomes less visible (which is the nice thing about constructor injection). But from an ease-of-use point of view, property injection is nicer.